templates/puppet/examples~credential.pp.erb

```diff
@@ -13,27 +13,29 @@
 # limitations under the License.
 <% end -%>
 <% if name != "README.md" -%>
-# Defines a credential to be used when communicating with Google Cloud
-# Platform. The title of this credential is then used as the 'credential'
-# parameter in the gdns_managed_zone type.
+# The following example requires two environment variables to be set:
+#     * cred_path - a path to a JSON service account file
+#     * project - the name of your GCP project
 #
-# For more information on the gauth_credential parameters and providers please
-# refer to its detailed documentation at:
-#
-#   https://forge.puppet.com/google/gauth
-#
-# For the sake of this example we set the parameter 'path' to point to the file
-# that contains your credential in JSON format. And for convenience this example
-# allows a variable named $cred_path to be provided to it. If running from the
-# command line you can pass it via Facter:
+# If running from the command line you can pass these via Facter:
 #
 #   FACTER_cred_path=/path/to/my/cred.json \
+#   FACTER_project='my-test-project'
 #       puppet apply <%= file_relative %>
 #
 # For convenience you optionally can add it to your ~/.bash_profile (or the
 # respective .profile settings) environment:
 #
 #   export FACTER_cred_path=/path/to/my/cred.json
+#   export FACTER_project='my-test-project'
+#
+# Defines a credential to be used when communicating with Google Cloud
```

Comment:
This sentence reads like it should have stayed towards the top of this section, or at least have something around it like "About credentials:"

Reply:
I like the idea of splitting this up into sections that way.

I wanted to make sure that the environment variable stuff was the first thing a user would read, since it's the most important.

I've split it into two sections: Getting Started + Authenticating with GCP